app/assets/javascripts/admin/addon/components/admin-plugin-config-metadata.gjs

```diff
@@ -0,0 +1,20 @@
+import i18n from "discourse-common/helpers/i18n";
```

Comment:
You can do this? I thought you were supposed to export the template as a constant

Reply:
This works too, but it's better to do it with a constant so the component has a name (useful in debugging/exploring with the Ember Inspector)

Reply:
Oh interesting, I was disappointed that ember inspector didn't give this a name, so I will do the const thing.

What happened here is that, I did something like:

```
export default <template>adssad</template>
```

And prettier autocorrected it to get rid of the export.

Reply:
Hmm I still get `unknownTemplateOnlyComponent` in the ember extension, even though I changed it to export in the same way as another template component I have here https://github.com/discourse/discourse/blob/5495279fb7995d4ff6ad57363437ccae60471262/app/assets/javascripts/admin/addon/components/admin-plugins-list.gjs . Not a huge deal, just seems unexpected.